autotest/interface/triton/test_triton_server_func.py

```diff
@@ -0,0 +1,201 @@
+        engine.triton_resume(session_id)
+        engine.triton_end(session_id)
+
+        engine.triton_set_session(None)
+        session_id = get_random_session_id()
+        engine.triton_infer(session_id, '你好，你叫什么名字' + str(2),
+                            'req_id:' + str(2), 50, False, True)
+        engine.triton_resume(session_id)
+        engine.triton_end(session_id)
+
+        engine.triton_set_session(None)
+        session_id = get_random_session_id()
+        engine.triton_infer(session_id, '你好，你叫什么名字' + str(3),
+                            'req_id:' + str(3), 50, False, False)
+        engine.triton_resume(session_id)
+        engine.triton_end(session_id)
+
+        engine.triton_set_session(None)
+        session_id = get_random_session_id()
+        engine.triton_infer(session_id, '你好，你叫什么名字' + str(4),
+                            'req_id:' + str(4), 50, True, True)
+        engine.triton_resume(session_id)
+        engine.triton_end(session_id)
+
+    # longinput
+    def test_long_input(self):
+        dict_info = {
+            'assistant': '<|Assistant|>െ',
+            'eoa': 'ി\n ',
+            'eoh': '\n ',
+            'eosys': '\n ',
+            'meta_instruction':
+            'You are an AI assistant whose name is InternLM (书生·浦语).',
+            'meta_tag': '<BOS>',
+            'model_name': 'puyu',
+            'repetition_penalty': 1.02,
+            'replace_token': 'ി',
+            'session_len': 8192,
+            'stop_words': ['ി'],
+            'system': '<|System|>െ',
+            'temperature': 1,
+            'top_k': 40,
+            'top_p': 0.8,
+            'user': '<|Human|>െ'
+        }
+
+        engine = Engine(SERVER_ADDR, log_level='ERROR', **dict_info)
+
+        session_id = get_random_session_id()
+        engine.triton_infer(session_id, '你好，你叫什么名字' * 10000,
+                            'req_id:' + str(1), 50, True, True)
+        engine.triton_resume(session_id)
+        engine.triton_end(session_id)
+
+    # diff dict
+    def test_diff_dict(server_addr):
+        with open('autotest/gen_case_dict.yaml') as f:
+            case_config = yaml.load(f.read(), Loader=yaml.SafeLoader)
+
+        try:
+            for case in case_config.keys():
+                print('dict case:' + case)
+                dict_info = case_config[case]
+                if dict_info is None:
+                    dict_info = {}
+
+                print('dict info:' + str(dict_info))
+                engine = Engine(server_addr, log_level='ERROR', **dict_info)
+
+                session_id = get_random_session_id()
+                engine.triton_infer(session_id, '你好，你叫什么名字', 'req_id:' + case,
+                                    500, True, True)
+                engine.triton_resume(session_id)
+                engine.triton_end(session_id)
+                engine.triton_set_session(None)
+        except Exception as e:
+            print(f'error: {e}')
```

Comment:
this testcase will alway pass and the error message only shows up if pytest runs with `-s`